client/src/components/Buttons.js

```diff
@@ -0,0 +1,47 @@
+import styled from 'styled-components'
```

Comment:
I got the recommendation lately to use 'styled-components/macro'. It adds a class with the corresponding name to each styled element and makes debugging really a little easier.

Reply:
Oh, I didn't know about this! Thank you for the recommendation! I will check it out and change accordingly! :D